site/htmltest.yaml

```diff
@@ -18,11 +18,13 @@ OutputDir: /go/src/agones.dev/agones/site/tmp
 OutputLogFile: .htmltest.log
 CheckExternal: true
 ExternalTimeout: 60
+HTTPConcurrencyLimit: 5
 IgnoreURLs:
   - http://localhost
   - https://twitter.com/agonesdev
   - https://www.youtube.com/playlist
   - https://docs.unrealengine.com
   - https://github.com/googleforgames/agones/blob/main/docs/proposals
+  - https://www.terraform.io/
```

Comment:
non blocking, but you could also `data-proofer-ignore` the link itself rather than skip it everywhere.